static/app/views/issueDetails/header.tsx

```diff
@@ -139,6 +139,13 @@ function GroupHeaderTabs({
       >
         {t('Merged Issues')}
       </TabList.Item>
+      <TabList.Item
+        key={Tab.RELATED_ISSUES}
+        hidden={organizationFeatures.has('organizations:related-issues')}
```

Comment:
```suggestion
        hidden={!organizationFeatures.has('related-issues')}
```

The feature flags in the frontend don't have the `organizations` prefix. Also you want to negate it here.